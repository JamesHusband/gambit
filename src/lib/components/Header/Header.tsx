import { Logo } from "../../ui";

export const Header = () => (
  <header className="fixed top-0 left-0 w-full p-4 z-10 bg-[#1a1a1a] border-b border-[#333]">
    <div className="flex items-center">
      <Logo />
    </div>
  </header>
);
