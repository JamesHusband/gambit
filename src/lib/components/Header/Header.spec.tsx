import { render, screen } from "@testing-library/react";
import { Header } from "./Header";

jest.mock("../../ui/Logo", () => ({
  Logo: () => <div>Logo</div>,
}));

describe("Header", () => {
  it("should render", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});

describe("Header", () => {
  it("should render", () => {
    render(<Header />);
    expect(screen.getByText("Logo")).toBeInTheDocument();
  });
});
