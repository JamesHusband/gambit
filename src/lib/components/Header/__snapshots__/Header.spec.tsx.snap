// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Header should render 1`] = `
<div>
  <header
    class="flex items-center justify-between px-6 py-4 bg-[#1a1a1a] border-b border-[#333]"
  >
    <div
      class="flex items-center"
    >
      <div>
        Logo
      </div>
    </div>
  </header>
</div>
`;
